pkg/traceql/parse_test.go

```diff
@@ -878,6 +878,7 @@ func TestAttributeNameErrors(t *testing.T) {
 		err error
 	}{
 		{in: "{ . foo }", err: newParseError("syntax error: unexpected END_ATTRIBUTE, expecting IDENTIFIER", 1, 3)},
+		{in: `{ . "foo" }`, err: newParseError("syntax error: unexpected END_ATTRIBUTE, expecting IDENTIFIER", 1, 3)},
```

Comment:
can we add a test for the error when a quote is opened and never closed? 

something like:
```
{ ."foo  }
```

Reply:
Added this. had to refactor the scope parsing piece a bit so that it returns expected error message